tests/test_api.py

```diff
@@ -345,44 +345,83 @@ def test_metadata_verify_delegate(self) -> None:
         with self.assertRaises(ValueError):
             role2.verify_delegate("role1", role1)
 
+    # pylint: disable=too-many-locals,too-many-statements
```

Comment:
I think this disables pylint downwards on class scope and not on function scope. Probably not what you intended, but also no big deal.